README.md

```diff
@@ -1,8 +1,11 @@
-# GovukTemplate
+# GOV.UK Template
 
 This provides a template containing the GOV.UK header and footer, and associated assets.
```

Comment:
Absolutely agree. That, and quite a lot of other stuff, did feel out of scope though.

I'll steal/add your eg line though 👍